demos/next-sdjwt-workshop/README.md

```diff
@@ -0,0 +1,49 @@
+<p align="center">
+  <a href="https://www.hyperledger.org/projects/identus">
+    <img src="https://cdn.jsdelivr.net/gh/hyperledger/identus@v2.13/resources/images/hyperledger-identus.svg" alt="identus-logo" width="513px" height="99px" />
+  </a>
+  <br>
+  <i> <font size="18">SDJWT Workshop Typescript</font> </i>
+  <br>
+</p>
+<hr>
+
+# Introduction
+This workshop will show you how to Issue and Verify a SD-JWT Credential and Presentation using connectionless flows, meaning, you won't need to establish a didcomm connection between the Issuer, Holder and Verifier.
+
+### **What Can I expect from this workshop?**
+
+You will learn everything that is needed to receive an Issued SD-JWT credential from the Cloud Agent and then use this credential to respond to a Presentation Submission request.
+
+### **What are connectionless flows?**
+
+In this presentation we won't be using [DIDComm Connections](https://hyperledger.github.io/identus-docs/docs/concepts/multi-tenancy#didcomm-connections) but instead we will generate Out of Band (OOB) codes for Issuance and Verification.
```

Comment:
What is this DIDComm Connections.
Is this the connection protocol that we had created? What are you talking about something else?